spec/grape_entity/entity_spec.rb

```diff
@@ -296,24 +296,13 @@ class Parent < Person
           expect(subject.exposures).to eq(name: {}, email: {})
         end
 
-        # the following 2 behaviors are testing because it is not most intuitive and could be confusing
-        context 'when  called from the parent class' do
-          it 'remove from parent and all child classes that have not locked down their attributes with an .exposures call' do
+        context 'when called from the parent class' do
+          it 'remove from parent and do not remove from child classes' do
             subject.expose :name, :email
             child_class = Class.new(subject)
             subject.unexpose :email
 
             expect(subject.exposures).to eq(name: {})
-            expect(child_class.exposures).to eq(name: {})
-          end
-
-          it 'remove from parent and do not remove from child classes that have locked down their attributes with an .exposures call' do
```

Comment:
from now, if you unexpose from parent class, child doesn't affect.

**master:**
```
[3] pry(main)> parent = Class.new(Grape::Entity)
=> #<Class:0x0000010250cf00>
[4] pry(main)> parent.expose :name, :email
=> [:name, :email]
[5] pry(main)> child = Class.new(parent)
=> #<Class:0x00000102a25938>
[6] pry(main)> child.exposures
=> {:name=>{}, :email=>{}}
[7] pry(main)> child_leaking = Class.new(parent)
=> #<Class:0x0000010292fab0>
[8] pry(main)> # not called .exposures
[9] pry(main)> parent.unexpose :email
=> {}
[10] pry(main)> child.exposures
=> {:name=>{}, :email=>{}}
[11] pry(main)> child_leaking.exposures
=> {:name=>{}}
```

**my-branch:**
```
[1] pry(main)> parent = Class.new(Grape::Entity)
NameError: uninitialized constant Grape
from (pry):1:in `__pry__'
[2] pry(main)> require 'grape-entity'
=> true
[3] pry(main)> parent = Class.new(Grape::Entity)
=> #<Class:0x000001031fb108>
[4] pry(main)> parent.expose :name, :email
=> [:name, :email]
[5] pry(main)> child = Class.new(parent)
=> #<Class:0x000001032a0888>
[6] pry(main)> child.exposures
=> {:name=>{}, :email=>{}}
[7] pry(main)> child_leaking = Class.new(parent)
=> #<Class:0x0000010319f588>
[8] pry(main)> # not called .exposures
[9] pry(main)> parent.unexpose :email
=> {}
[10] pry(main)> child.exposures
=> {:name=>{}, :email=>{}}
[11] pry(main)> child_leaking.exposures
=> {:name=>{}, :email=>{}}
```

It was introduces in f089f16e and I think comment 

```
# the following 2 behaviors are testing because it is not most intuitive and could be confusing
```

explains why. Because exposures merge happened in children classes only when it was called.

I'll add entry to `UPGRADING`.